djangocms_snippet/models.py

```diff
@@ -72,7 +73,8 @@ class SnippetPtr(CMSPlugin):
         on_delete=models.CASCADE,
     )
 
-    snippet = models.ForeignKey(Snippet, on_delete=models.CASCADE,)
+    snippet = models.ForeignKey(Snippet, on_delete=models.CASCADE)
+    new_snippet = models.ForeignKey(SnippetGrouper, on_delete=models.CASCADE)
```

Comment:
This field replaces the old, although to aid backward compatibility we should probably have a property that returns the latest published snippet when accessing snippet.

```suggestion
    snippet_grouper = models.ForeignKey(SnippetGrouper, on_delete=models.CASCADE)
```

```
@property
def snippet(self):
    return snippet_grouper.snippet.first()
```